packages/e2e-test-utils/src/theme-installed.js

```diff
@@ -12,12 +10,10 @@ import { switchUserToTest } from './switch-user-to-test';
  * @return {boolean} Whether the theme exists.
  */
 export async function isThemeInstalled( slug ) {
-	await switchUserToAdmin();
-	await visitAdminPage( 'themes.php' );
-
-	await page.waitForSelector( 'h2', { text: 'Add New Theme' } );
-	const found = await page.$( `[data-slug="${ slug }"]` );
-
-	await switchUserToTest();
-	return Boolean( found );
+	const themes = await rest( {
+		method: 'GET',
+		path: '/wp/v2/themes',
```

Comment:
We might need to pass `per_page=-1` to get all the themes (it defaults to 10).

Reply:
We can also use the `search` param - https://developer.wordpress.org/rest-api/reference/themes/#arguments